src/components/layout/nav/ToolbarMenu.tsx

```diff
@@ -16,10 +16,10 @@ export const TOOLBAR_MENU_ITEMS: (Required<
   })[] = [
   {
     permissions: ['canViewMyDashboard'],
-    path: Routes.MY_DASHBOARD,
-    id: 'navToMyDashboard',
-    activeItemPathIncludes: 'my-dashboard',
-    title: 'My Dashboard',
+    path: Routes.USER_DASHBOARD,
+    id: 'navToUserDashboard',
+    activeItemPathIncludes: 'dashboard',
```

Comment:
I'm not seeing the nav item be in an active state when I'm on the dashboard:
<img width="1447" alt="Screenshot 2025-05-23 at 1 16 45 PM" src="https://github.com/user-attachments/assets/aa65f402-1716-4fdb-bf86-a7abc2ab5619" />